src/app/feature/map/search/search.component.ts

```diff
@@ -39,78 +26,74 @@ export class SearchComponent implements OnInit {
   constructor(
     private readonly _mapService: MapService,
     private readonly _mapDataProviderService: MapDataProviderService,
-    private readonly fb: FormBuilder,
     private readonly translateService: TranslateService
   ) {
-    this.stateGroups = [];
+    this.destinationGroups = [];
+    this.userLocationGroups = [];
   }
 
-  @Output() public setLocation = new EventEmitter<LocationNode>();
-
-  public locations: GuideMapFeaturePoint[];
-
-  public stateForm: FormGroup = this.fb.group({
-    stateGroup: '338',
-  });
+  @Output() set location(value: string) {
```

Comment:
Looks like we don`t need   @Output() set location(value: string) {
    value && value.length && this.findUserLocation(value);
  }
  @Output() set destination(value: string) {
    value && value.length && this.findDestination(value);
  }
code anymore